src/test/resources/Tests/arm/component/ArchiveAnnotationMetadata/expectedResponse.a360

```diff
@@ -1,2 +1,35 @@
-{"operation":"create_record","relation_id":"1234","record_metadata":{}}
-{"operation":"upload_new_file","relation_id":"1234","file_metadata":{}}
+{
+  "operation": "create_record",
+  "relation_id": "1234",
+  "record_metadata": {
+    "record_class": "DARTS",
+    "publisher": "DARTS",
+    "region": "GBR",
+    "recordDate": "2024-01-23T10:00:00",
```

Comment:
There's something strange with `recordDate` here and throughout these expected responses.
It's missing the UTC 'Z' timezone bit like elsewhere in this file e.g. `event_date`.
Please can you check the ISO date-time string